vcpkg/consume/manifest-mode.md

```diff
@@ -0,0 +1,331 @@
+ValidateSolutionConfiguration:
+  Building solution configuration "Debug|x64".
+Project "D:\projects\manifest-example\manifest-example.sln" (1) is building "D:\projects\manifest-example\manifest-example.vcxproj" (2) on node 1 (default targets).
+PrepareForBuild:
+  (omitted)
+InitializeBuildStatus:
+  (omitted)
+ComputeStdModulesCompileInputs:
+  (omitted)
+SetModuleDependencies:
+  Creating directory "x64\Debug\manifest.ceffc6eb_MD.tlog\".
+VcpkgTripletSelection:
+  Using triplet "x64-windows" from "D:\projects\manifest-example\vcpkg_installed\x64-windows\x64-windows\"
+  Using normalized configuration "Debug"
+VcpkgInstallManifestDependencies:
+  Installing vcpkg dependencies to D:\projects\manifest-example\vcpkg_installed\x64-windows\
+  Creating directory "D:\projects\manifest-example\vcpkg_installed\x64-windows\".
+  "D:\vcpkg\vcpkg.exe" install  --x-wait-for-lock --triplet "x64-windows" --vcpkg-root "D:\vcpkg\" "--x-manifest-root=D:\projects\manifest-example\" "--x-install-root=D:\projects\manifest-example\vcpkg_installed\x64-windows\"
+  "D:\vcpkg\vcpkg.exe" install  --x-wait-for-lock --triplet "x64-windows" --vcpkg-root "D:\vcpkg\" "--x-manifest-root=D:\projects\manifest-example\" "--x-install-root=D:\projects\manifest-example\vcpkg_installed\x64-windows\"
+  Detecting compiler hash for triplet x64-windows...
+  The following packages will be built and installed:
+      cxxopts:x64-windows -> 3.1.1
+      fmt:x64-windows -> 10.0.0
+      range-v3:x64-windows -> 0.12.0#1
+    * vcpkg-cmake:x64-windows -> 2023-05-04
+    * vcpkg-cmake-config:x64-windows -> 2022-02-06#1
+  (omitted)
+ClCompile:
+  (omitted)
+Link:
+  (omitted)
+AppLocalFromInstalled:
+  pwsh.exe -ExecutionPolicy Bypass -noprofile -File "D:\vcpkg\scripts\buildsystems\msbuild\applocal.ps1" "D:\projects\manif
+  est-mode-msbuild\x64\Debug\manifest-example.exe" "D:\projects\manifest-example\vcpkg_installed\x64-windows\x64-windows\debug\bin"
+  "x64\Debug\manifest.ceffc6eb.tlog\manifest-example.write.1u.tlog" "x64\Debug\vcpkg.applocal.log"
+  D:\projects\manifest-example\x64\Debug\fmtd.dll
+FinalizeBuildStatus:
+  (omitted)
+Done Building Project "D:\projects\manifest-example\manifest-example.vcxproj" (default targets).
+
+Done Building Project "D:\projects\manifest-example\manifest-example.sln" (default targets).
+
+Build succeeded.
+```
+
+### [Visual Studio](#tab/build-visual-studio)
+
+By default, manifest mode is disabled in Visual Studio projects. To enable manifests, open the
+project's Properties window and set `Use Vcpkg Manifest` to "Yes" in the vcpkg tab.
+
+:::image type="complex" source="../resources/vs-enable-vcpkg-manifest.png" alt-text="Screenshot of a Visual Studio's project Properties window.":::
+Screenshot of a Visual Studio project properties window. The VCPKG tab is selected and highlighted. The
+option named "use VCPKG manifest" is highlighted with its value set to "Yes".
```

Comment:
```suggestion
Screenshot of a Visual Studio project properties window. The vcpkg tab is selected and highlighted. The
option named "Use Vcpkg Manifest" is highlighted with its value set to "Yes".
```

Reply:
The capitalization here was deliberate to help screen readers pronounce vcpkg correctly as per the guidelines.